website/source/community.html.erb

```diff
@@ -37,6 +37,7 @@ description: |-
 <ul>
   <li><a href="https://github.com/iverberk/nomad-ui">nomad-ui</a> - Nomad UI is a simple to deploy, web based UI for interacting with Nomad</li>
```

Comment:
oh, this link is wrong :o 

and missing awesome-nomad too :)

Reply:
@jippi could you send a PR fixing these up. You obviously know the links :)